packages/features/portfolio/src/portfolio-feature-shell.tsx

```diff
@@ -189,6 +190,9 @@ const Portfolio = observer(() => {
             {params.category === PORTFOLIO_CATEGORIES.TRANSACTION_HISTORY && (
               <PortfolioTransactionHistory />
             )}
+            {params.category === PORTFOLIO_CATEGORIES.RISK && (
```

Comment:
Can you also change the icon for the risk tab to this one:
https://www.figma.com/design/FpS9IqY9TDNHcubYoBkAIH/Dev-File?node-id=14622-11892&t=xKYtw2o0blPnkXRJ-1